NGS-QC/tutorials/dive_into_qc.md

```diff
@@ -52,21 +52,29 @@ Short introduction about this subpart.
 
 ## Check the FastQC output
 
-Short introduction about this subpart.
+FastQC outputs are an html report and a raw data file. If the raw data file allows the user to start from original data to create its own grpahs and stats and the html report shows directly a graphical web page, the two outputs are divided in sections :
 
-:pencil2: ***Hands on!***
+:pencil2: ***Basics statistics***
```

Comment:
The "Hands on" must stay and after we would like tasks to do. If you want a subsection, use `### Basics statistics`